aws-ts-ec2-provisioners/util.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright 2016-2020, Pulumi Corporation.  All rights reserved.
+
+import * as crypto from "crypto";
+import * as fs from "fs";
+import * as path from "path";
+
+/**
+ * Calculates a hash for all of the files under the scripts directory.
+ */
+export function getFileHash(filename: string): string {
+	const data = fs.readFileSync(path.join(__dirname, filename), {encoding: "utf8"});
```

Comment:
FWIW - I think the current interpretation of `src: ` is relative to cwd, not to the `__dirname`.

Reply:
I am not sure I understand this comment?